### DISCUSSION TITLE
Referencing data stored in an external table when filtering.

### DISCUSSION THREAD

**Discussion by enterrift76** [supabase/supabase#21105] [Questions]
Good day,

I have two tables in my app, "vehicles" and "drivers". When filtering my vehicles table for a specific vehicle I get my primary key "driverID" returned for the driver value. Is it possible to also return data from the driver's table such as driver's full name using a filter on the Vehicle table? I know that when performing a GET we would use notation similar to this: 

https://apiurl.com/?select=*,driver(Fullname)

**Comment by GaryAustin1** [comment]
Not really clear what you are asking.  But if it is can you filter on a table linked by a foreign key to the main table yes.
You are showing using the URL format directly.  If that is how you are working, versus a Supbase client, then this reference is really where you should go: https://postgrest.org/en/stable/references/api/resource_embedding.html#embedded-filters
PostgREST is the API Supabase uses for database access.
